TheMovieDB/TheMovieDB/Presentation/Coordinator/Coordinator.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Coordinator.swift
+//  TheMovieDB
+//
+//  Created by Agustin Godoy Cosser on 12/10/2022.
+//
+
+import Foundation
+import UIKit
+
+protocol Coordinator {
+    var childCoordinator: [Coordinator] { get set }
+    var navigationController: UINavigationController { get set }
+    
+    func start()
+}
```

Comment:
should be inside a Protocol folder

Reply:
https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d